bin/review-rot

```diff
@@ -80,26 +85,74 @@ def main(cli_args, valid_choices):
                         state_=arguments.get('state'),
                         value=arguments.get('value'),
                         duration=arguments.get('duration'),
+                        last_commented=arguments.get('last_commented'),
                         token=token,
                         host=item.get('host'),
                         ssl_verify=arguments.get('ssl_verify'),
                     )
                 )
 
     # Now, with all results in place, sort them and print
+    if arguments.get('comment_sort'):
+        sorting_key = sort_by_last_comment
+    else:
+        sorting_key = operator.attrgetter('time')
+
     sorted_results = sorted(
         results,
-        key=operator.attrgetter('time'),
+        key=sorting_key,
         reverse=arguments.get('reverse'),
     )
-
     formatting = arguments.get('format', 'oneline')
 
-    n = len(results)
-    print(report_prefixes[formatting])
-    for i, result in enumerate(sorted_results):
-        print(result.format(style=formatting, i=i, N=n))
-    print(report_suffixes[formatting])
+    if not arguments.get('email'):
+        n = len(results)
+        print(report_prefixes[formatting])
+        for i, result in enumerate(sorted_results):
+            print(result.format(style=formatting, i=i, N=n,
+                  last_commented=arguments.get('last_commented')))
+        print(report_suffixes[formatting])
+
+    else:
+        log.debug('SENDING MAIL')
+
+        loader = FileSystemLoader(
+            searchpath=pkg_resources.resource_filename('reviewrot', ''))
+        env = Environment(loader=loader)
+        env.filters['formatduration'] = format_duration
+
+        TEMPLATE_FILE = 'html_template.jinja'
```

Comment:
if this is a constant it should be declared outside this function

Reply:
@alexandrevicenzi